source/extensions/filters/network/zookeeper_proxy/filter.cc

```diff
@@ -216,13 +244,53 @@ void ZooKeeperFilter::onDecodeError() {
   setDynamicMetadata("opname", "error");
 }
 
-void ZooKeeperFilter::onRequestBytes(const uint64_t bytes) {
+void ZooKeeperFilter::onDefaultRequestBytes(const OpCodes opcode, const uint64_t bytes) {
+  auto it = config_->op_code_map_.find(opcode);
```

Comment:
is it possible that the opcode is not in the map? if it's impoosible, assert and log error